build.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+# Correct Syntax: ./build.sh [port [*upload | verify]]
+set -e
+HERE=$(pwd)
+VERSION='Voxel8 Marlin Build Script v1.0'
+
+# Begin build script
+if [ -z "$1" ]; then
+  echo "Please enter the port of the device or specify 'verify'"
+  exit 1
+fi
+if [ $1 = "upload" ]; then
+  echo "Syntax error. Please use --help for more info."
```

Comment:
This happens when upload is specified without a port, correct? If so we should make the error message reflect that.